src/example-relay/src/LocalForm/index.js

```diff
@@ -94,7 +94,10 @@ export default function LocalForm() {
         environment={environment}
         query={graphql`
           query LocalFormQuery {
-            __typename # https://github.com/facebook/relay/issues/2471
+            # Should be ok, we are not calling the BE anyway
```

Comment:
The comment `https://github.com/facebook/relay/issues/2471` should stay there - it's not related to this change. Also see: https://github.com/facebook/relay/commit/793729e7af9c7ee0de971e3d2ed26e5896774640#commitcomment-37652508

Reply:
Comment is back in 🙂